frontend/packages/dev-console/src/components/PodsRing.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { VictoryLabel } from 'victory';
+import * as _ from 'lodash';
+import {
+  K8sResourceKind,
+  k8sPatch,
+  K8sKind,
+  SelfSubjectAccessReviewKind,
+} from '@console/internal/module/k8s';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import { checkEditAccess } from '@console/internal/components/utils';
+import PodStatus from './topology/shapes/PodStatus';
```

Comment:
`PodStatus` needs to be refactored out of dev-console because we do not want direct dependencies on dev-console from admin-console.
There is a `@console/shared` package to hold common components.